File: 30-Days-of-Code/Day 26 - Nested Logic/Solution.java
import java.io.*;
import java.util.*;
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        
        Scanner in = new Scanner(System.in);
        
        int iDay = in.nextInt();
        int iMonth = in.nextInt();
        int iYear = in.nextInt();
        
        int fDay = in.nextInt();
        int fMonth = in.nextInt();
        int fYear = in.nextInt();
        
        int fine = 0;
        
        if (iYear == fYear) {
            if (iMonth > fMonth) {
                fine = (iMonth - fMonth) * 500;
            } else if (iMonth == fMonth && iDay > fDay){
                fine = (iDay - fDay) * 15;
            } 
        } else if (iYear > fYear) fine = 10000;
        
        System.out.println(fine);
    }
}

File: 30-Days-of-Code/Day 26 - Nested Logic/Solution.py
# Enter your code here. Read input from STDIN. Print output to STDOUT

iDay, iMonth, iYear = [int(x) for x in input().split(' ')]
fDay, fMonth, fYear = [int(x) for x in input().split(' ')]

fine = 0

if iYear > fYear:
    fine = 10000;
elif iYear == fYear:
    if iMonth > fMonth:
        fine = (iMonth - fMonth) * 500
    elif iMonth == fMonth and iDay > fDay:
        fine = (iDay - fDay) * 15
        
print(fine)